resources/services/apigatewayv2/apis.go

```diff
@@ -465,7 +465,7 @@ func Apigatewayv2Apis() *schema.Table {
 								Name:        "arn",
 								Description: "The Amazon Resource Name (ARN) for the resource.",
 								Type:        schema.TypeString,
-								Resolver: client.ResolveARN(client.ApigatewayService, func(resource *schema.Resource) ([]string, error) {
+								Resolver: client.ResolveARNWithRegion(client.ApigatewayService, func(resource *schema.Resource) ([]string, error) {
 									r := resource.Item.(types.IntegrationResponse)
 									api := resource.Parent.Parent.Item.(types.Api)
 									return []string{apiIDPart, *api.ApiId, "integrationresponses", *r.IntegrationResponseId}, nil
```

Comment:
I think it should be
`
return []string{apiIDPart, *api.ApiId, "integrations", *i.IntegrationID, "integrationresponses", *r.IntegrationResponseId}, nil
`

Reply:
Ouch. Fixed.